esmvaltool/cmorizers/obs/cmorize_obs_era_interim.py

```diff
@@ -0,0 +1,329 @@
+    attributes['mip'] = var['mip']
+    cmor_table = CMOR_TABLES[attributes['project_id']]
+    definition = cmor_table.get_variable(var['mip'], var['short_name'])
+
+    with catch_warnings():
+        filterwarnings(
+            action='ignore',
+            message="Ignoring netCDF variable 'tcc' invalid units '(0 - 1)'",
+            category=UserWarning,
+            module='iris',
+        )
+        filterwarnings(
+            action='ignore',
+            message="Ignoring netCDF variable 'lsm' invalid units '(0 - 1)'",
+            category=UserWarning,
+            module='iris',
+        )
+        filterwarnings(
+            action='ignore',
+            message=("Ignoring netCDF variable 'e' invalid units "
+                     "'m of water equivalent'"),
+            category=UserWarning,
+            module='iris',
+        )
+        if 'files' in var:
+            if var['operator'] == 'sum':
+                # Multiple variables case using sum operation
+                for i, item in enumerate(in_file):
+                    in_cube = iris.load_cube(
+                        str(item),
+                        constraint=utils.var_name_constraint(var['raw'][i]),
+                    )
+                    if i == 0:
+                        cube = in_cube
+                    else:
+                        cube += in_cube
+                cube.var_name = var['short_name']
+        else:
+            cube = iris.load_cube(
+                str(in_file),
+                constraint=utils.var_name_constraint(var['raw']),
+            )
+
+    _set_global_attributes(cube, attributes, definition)
+
+    # Set correct names
+    cube.var_name = definition.short_name
+    cube.standard_name = definition.standard_name
+    cube.long_name = definition.long_name
+
+    # Fix data type
+    cube.data = cube.core_data().astype('float32')
+
+    _fix_coordinates(cube, definition)
+    cube = _fix_frequency(cube, var)
+
+    # Convert units if required
+    cube.convert_units(definition.units)
+
+    # Make latitude increasing
+    cube = cube[:, ::-1, ...]
+
+    logger.info("Saving cube\n%s", cube)
+    logger.info("Expected output size is %.1fGB",
+                np.prod(cube.shape) * 4 / 2 ** 30)
+    utils.save_variable(
+        cube,
+        cube.var_name,
+        out_dir,
+        attributes,
+        local_keys=['positive'],
+    )
+
+
+def cmorization(in_dir, out_dir, cfg, _):
+    """Cmorization func call."""
+    cfg['attributes']['comment'] = cfg['attributes']['comment'].format(
+        year=datetime.now().year)
+    cfg.pop('cmor_table')
+
+    n_workers = int(cpu_count() / 1.5)
+    logger.info("Using at most %s workers", n_workers)
+    futures = {}
+    with ProcessPoolExecutor(max_workers=1) as executor:
+        for short_name, var in cfg['variables'].items():
+            if 'short_name' not in var:
+                var['short_name'] = short_name
+            if 'file' in var:
```

Comment:
Please avoid the code duplication below

Reply:
@bouweandela Thanks, the duplication below is removed.